src/main/java/com/redhat/devtools/lsp4ij/settings/ui/JsonTextField.java

```diff
@@ -76,6 +81,7 @@ public void setJsonFilename(@NotNull String jsonFilename) {
             Document document = editorTextField.getDocument();
             VirtualFile file = FileDocumentManager.getInstance().getFile(document);
             if (file != null) {
+                file.rename(this, "foo.json");
```

Comment:
What's going on here? Why is this renaming and then immediately renaming again?

Reply:
Good catch, removed.